### DISCUSSION TITLE
bug? dragging causes tab to jump to bottom

### DISCUSSION THREAD

**Discussion by Elementari** [mbnuqw/sidebery#1610] [General]
Hello, recently I noticed that when I drag a tab, e.g. downward 2 cm to another position a few tabs further down, the tab will sometimes NOT be placed there, but jump to the very BOTTOM of the complete tap list. Note that I do not have an separation lines between tabs. But the issues was not present in v4, and I think it was also not present some time ago in v5. 
I happens in about 40% of situations! I cannot reproduce it, it seems random.

**Comment by cdavie-artium** [comment]
I'm also not able to drag tabs to wherever I want. They often jump to the bottom, like @Elementari reports.
